config/mongoid.yml

```diff
@@ -9,7 +10,7 @@ development:
       # Provides the hosts the default session can connect to. Must be an array
       # of host:port pairs. (required)
       hosts:
-        - mongodb:27017
+        - localhost:27017
```

Comment:
I think this was  a change you had to make to get your dev environment working, but I think we shouldn't include this change in this pull request. 

To do that, you'll need to "unstage" this file and force push the commit. See [here](https://stackoverflow.com/posts/38744242/revisions) for an example!

Reply:
@toomuchpete Yeah, made the changes.